.github/workflows/rtm.yml

```diff
@@ -6,6 +6,7 @@ on:
 jobs:
   cmd:
     name: Ready To Merge
+    if: github.event_name == 'pull_request'
```

Comment:
Actually I am not 100% sure if that works here within a reusable workflow. I tought I did experience some problems, but maybe I just remember incorrectly or it was something else...

Reply:
OK, I was remembering wrong, this will work, we already use other if conditions in the reusable workflow. Got confused.